docs/standard/serialization/binary-serialization.md

```diff
@@ -22,351 +22,329 @@ When implementing a serialization mechanism in an object-oriented environment, y
 
 [!INCLUDE [binary-serialization-warning](../../../includes/binary-serialization-warning.md)]
 
-As the nature of binary serialization allows the modification of private members inside an object and therefore changing the state of it, other serialization frameworks like JSON.NET which operate on the public API surface are recommended.
+Binary serialization allows modifying private members inside an object and therefore changing the state of it. Because of this, other serialization frameworks, like JSON.NET, that operate on the public API surface are recommended.
```

Comment:
The rewrite preserves the original meaning, but this paragraph doesn't make sense to me. I don't see the causal link between sentence 1 and sentence 2. And on the surface it looks problematic since JSON.NET isn't a binary serialization framework.

In any case, anyplace we were recommending JSON.NET we should change to System.Text.Json.

```suggestion
Binary serialization allows modifying private members inside an object and therefore changing the state of it. Because of this, other serialization frameworks, like <xref:System.Text.Json>, that operate on the public API surface are recommended.
```